core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -133,7 +133,8 @@ trait EndToEndAuthorizationTest extends IntegrationTestHarness with SaslSetup {
   this.serverConfig.setProperty(KafkaConfig.AuthorizerClassNameProp, classOf[SimpleAclAuthorizer].getName)
   // Some needed configuration for brokers, producers, and consumers
   this.serverConfig.setProperty(KafkaConfig.OffsetsTopicPartitionsProp, "1")
-  this.serverConfig.setProperty(KafkaConfig.OffsetsTopicReplicationFactorProp, "1")
+  this.serverConfig.setProperty(KafkaConfig.OffsetsTopicReplicationFactorProp, "3")
```

Comment:
misconfigured doesn't seem wrong, because with the offset topic having a replication factor = 1, and the server being configured with min.insync = 3, no messages can be sent with ack=-1 which is the default for the offset topic.